### DISCUSSION TITLE
3D games can see the light?

### DISCUSSION THREAD

**Discussion by ismailkayzer** [skyline-emu/skyline#1464] [General]
I am looking forward to an update that supports 3D games without any lagging or black screen issues, such as FIFA and Sniper Elite. Thank you for your all the efforts you're putting into this emulator.

**Comment by Joemama45** [comment]
> I am looking forward to an update that supports 3D games without any lagging or black screen issues, such as FIFA and Sniper Elite. Thank you for your all the efforts you're putting into this emulator.

You can test out clustertruck, it's the only game that skyline supports, but it doesn't or do save your progress when you stop playing

**Reply by ismailkayzer** [reply]
thanks I'll try it

**Comment by Crytonics** [comment]
You can check games list here: https://github.com/skyline-emu/skyline-games-list/issues
